src/frontends/onnx/frontend/src/utils/common.cpp

```diff
@@ -141,6 +142,44 @@ bool is_optimized_out(const Output<ov::Node>& node_output) {
     return rt_info.find(OPTIMIZED_OUT_NODE) != rt_info.end();
 }
 
+std::string collect_translation_exceptions(const std::shared_ptr<ov::Model>& partially_converted) {
+    std::string fully_unsupported_ops = "OpenVINO does not support the following ONNX operations: ";
+    std::string additional_error_message = "Errors during ONNX translation: \n";
+    const std::string sep = ", ";
+
+    bool unsupported_found = false;
+    bool additional_error_found = false;
+    for (const auto& op : partially_converted->get_ops()) {
+        if (const auto unsupported = std::dynamic_pointer_cast<frontend::NotSupportedONNXNode>(op)) {
+            if (unsupported->additional_error_message().empty()) {
+                fully_unsupported_ops += (unsupported->get_attrs().get_opset_name().empty()
+                                              ? ""
+                                              : unsupported->get_attrs().get_opset_name() + ".") +
+                                         unsupported->get_attrs().get_type_name() + sep;
+                unsupported_found = true;
+            } else {
+                additional_error_message += unsupported->additional_error_message();
+                additional_error_found = true;
+            }
+        }
+    }
+    fully_unsupported_ops = fully_unsupported_ops.substr(0, fully_unsupported_ops.size() - sep.size());
+    // remove redundant new line
+    additional_error_message =
+        (additional_error_message.empty() || additional_error_message[additional_error_message.length() - 1] != '\n')
+            ? additional_error_message
+            : additional_error_message.erase(additional_error_message.length() - 1);
+    if (unsupported_found && additional_error_found) {
+        return fully_unsupported_ops + "\n" + additional_error_message;
+    } else if (unsupported_found) {
+        return fully_unsupported_ops;
+    } else if (additional_error_found) {
+        return additional_error_message;
+    } else {
+        return "";
+    }
```

Comment:
Would that work?
```suggestion
    const auto separator = fully_unsupported_ops && additional_error_message ? "\n" : "";
    return fully_unsupported_ops + separator + additional_error_message;
```

Reply:
Not really, `fully_unsupported_ops` and `additional_error_message` have initial values:
```
  std::string fully_unsupported_ops = "OpenVINO does not support the following ONNX operations: ";
  std::string additional_error_message = "Errors during ONNX translation: \n";
```
which should be skipped if not used